src/db.rs

```diff
@@ -131,16 +133,19 @@ impl Deref for DieselPooledConn<'_> {
     }
 }
 
-pub fn connect_now() -> ConnectionResult<PgConnection> {
-    let url = connection_url(&crate::env("DATABASE_URL"));
+pub fn connect_now(config: &config::Server) -> ConnectionResult<PgConnection> {
+    let url = connection_url(config, &config.db.primary.url);
     PgConnection::establish(&url)
 }
 
-pub fn connection_url(url: &str) -> String {
+pub fn connection_url(config: &config::Server, url: &str) -> String {
     let mut url = Url::parse(url).expect("Invalid database URL");
-    if dotenv::var("HEROKU").is_ok() && !url.query_pairs().any(|(k, _)| k == "sslmode") {
+
+    // Enforce secure connections in production.
+    if config.base.env == Env::Production && !url.query_pairs().any(|(k, _)| k == "sslmode") {
```

Comment:
seems a bit unfortunate for the database code to have to be aware of all the other server config values. what do you think about decoupling this a bit and adding something like `sslMode` to the database config struct instead?

Reply:
I don't see much of a problem personally of passing the whole configuration, but I can refactor things to pass only the database configuration.

Reply:
Pushed a new commit to request only the database pools config instead of the whole server config. Let me know if this is what you intended here :slightly_smiling_face: